misc/host_tests.conf

```diff
@@ -12,6 +12,7 @@ add mudgee
 
 # Additional base modules
 include subset/pentests/build.conf
+include subset/bacnet/build.conf
```

Comment:
Should not be here -- this is for baseline tests only. Should be in all_tests.conf